<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memberes</class>
 <widget class="QDialog" name="memberes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#1b1a1c;
</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-size:18px;background-color:#6025d1;
color:#fff;
border-radius:12px;</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;background-color:#6025d1;
color:#fff;
border-radius:12px;</string>
   </property>
   <property name="text">
    <string>MODIFIER MEMBERE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>330</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;background-color:#6025d1;
color:#fff;
border-radius:12px;
</string>
   </property>
   <property name="text">
    <string>FICHIE MEMBERE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>330</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;
background-color:#6025d1;
color:#fff;
border-radius:12px;
</string>
   </property>
   <property name="text">
    <string>RESTAURE MEMBERE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>330</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;background-color:#6025d1;
color:#fff;
border-radius:12px;</string>
   </property>
   <property name="text">
    <string>SUPPRIMER MEMBERE</string>
   </property>
  </widget>
  <widget class="QLabel" name="plus">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#6025d1;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plus_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#6025d1;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plus_3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>330</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#6025d1;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plus_4">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>330</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#6025d1;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="plus_5">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>330</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#6025d1;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#dd4b3a;
color:#fff;
border-radius:12px;</string>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
